css/index.css

```diff
@@ -77,14 +80,16 @@ header {
 
 header nav {
     margin-top: 40px; 
-    width: 610px;
+    width: 69.318%; /*610px69.318*/
```

Comment:
I really like that you are using comments to show the changes you made from px to %.  This will be very beneficial for your learning progress.